xbmc/pvr/PVRManager.cpp

```diff
@@ -627,15 +627,16 @@ bool CPVRManager::Load(void)
   /* start the add-on update thread */
   if (m_addons)
     m_addons->Start();
+  else
+    return false;
 
-  /* load at least one client */
-  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
-    Sleep(50);
```

Comment:
why is this so urgent that it needs a half fix that may or may not be 
fixed properly at some point? you're removing the "one add-on loaded" 
code now, which was there on purpose, which used to work a couple of 
releases back and which was broken while refactoring a while ago.

unless there's a very good reason to not fix this properly now 
(shouldn't be that much work), it gets a -1 from me at this point in the 
release schedule.

On 30-04-15 12:26, Sam Stenvall wrote:
> In xbmc/pvr/PVRManager.cpp
> <https://github.com/xbmc/xbmc/pull/7020#discussion_r29418339>:
>
>>
>> -  /* load at least one client */
>> -  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
>> -    Sleep(50);
>
> I specifically stated in the PR description that I know this is not the
> best way but that we need some kind of fix for the next release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/7020/files#r29418339>.
>